### PR TITLE
migrate to metatests@0.2.0

### PR DESCRIPTION
**Pull request by lundibundi**
Finished migration.

### CODE REVIEW

File: test/all.js [hunk 1]
```diff
@@ -1,22 +1,8 @@
...
-  'safe', 'sort', 'strings', 'time', 'units', 'zip', 'zipWith',
-  'iterator', 'utilities'
-];
-
-all.forEach(name => require('./' + name));
-
-api.metatests.report();
+fs.readdir(__dirname, (err, files) => {
+  if (err) return console.error('Failed to run tests: ', err);
+  files.forEach(name => require('./' + name));
```
Comment (nechaido) [inline test/all.js]: I'd prefer to filter out the current file.

**Comment by lundibundi** [comment]
@nechaido @belochub PTAL, I want to merge this ASAP.

File: test/all.js [hunk 2]
```diff
@@ -1,22 +1,9 @@
...
-  'safe', 'sort', 'strings', 'time', 'units', 'zip', 'zipWith',
-  'iterator', 'utilities'
-];
-
-all.forEach(name => require('./' + name));
-
-api.metatests.report();
+fs.readdir(__dirname, (err, files) => {
+  if (err) return console.error('Failed to run tests: ', err);
+  files.filter(file => file !== 'all.js')
```
Comment (belochub) [inline test/all.js]: I don't like the idea of hard coding the current file's name.
Can you use `__filename` here, please?

Comment (lundibundi) [reply test/all.js]: Oh, yeah, thanks.

**Comment by lundibundi** [comment]
Landed in bca504784c2f77ce58086072f4271c496a83f30f.

**Pull request closed without merging**